pkg/prometheus/operator.go

```diff
@@ -213,9 +218,13 @@ func New(conf Config, logger log.Logger) (*Operator, error) {
 		listwatch.MultiNamespaceListerWatcher(c.config.Namespaces, func(namespace string) cache.ListerWatcher {
 			return &cache.ListWatch{
```

Comment:
There is already a type for filtered list watches, we should re-use that: https://github.com/kubernetes/client-go/blob/7b18d6600f6b0022e31c46b46875beffd85cc71a/tools/cache/listwatch.go#L81

Reply:
Interesting.

it doesn't use typed clients, can it be a problem? 

also what would be a resource name? just "prometheus"? , but where apiversion is specified then?

Reply:
hmm you're right .. doesn't look like we can easily wrap this, given those circumstances what you did here lgtm